sgpc3_with_shtc1/README.md

```diff
@@ -0,0 +1,8 @@
+# SGPC3 with SHTC1: A Humidity-compensated Combo Driver for SGPC3
+This folder contains a humidity compensated combo-driver combining the SHTC3 and
+the SHTC1 (family compatible like SHTW1, SHTW2, SHTC3) driver into one with the
```

Comment:
I'd either add SHTC1 to the list as well or leave of SHTW1 since it is a legacy device.

Reply:
done